### ISSUE TITLE
建议plugin.xml修改下

### ISSUE THREAD

**Issue by likyh** [NiklasMerz/cordova-plugin-fingerprint-aio#93]
```
  <!-- android -->
  <platform name="android">
    <config-file target="app/src/main/res/xml/config.xml" parent="/*">
      <feature name="Fingerprint">
        <param name="android-package" value="de.niklasmerz.cordova.fingerprint.Fingerprint"/>
      </feature>
    </config-file>
    <config-file target="AndroidManifest.xml" parent="/*">
      <uses-permission android:name="android.permission.USE_FINGERPRINT"/>
    </config-file>
```

建议改成这样

**Comment by Liam-Ryan** [comment]
This change fixes my issue with android ( #92 )

**Comment by NiklasMerz** [comment]
Thank you for this code snippet. I will check this for the pull requests.
